software/services/employee/app/routers/cv_management.py

```diff
@@ -0,0 +1,24 @@
+from fastapi import APIRouter, HTTPException
+from models import cv
+
+router = APIRouter()
```

Comment:
Fajną praktyką jest dodawanie takich meta-informacji o endpointach, bo fajnie się to samo dokumentuje w Swagger'rze. Ale generalnie wszystko git.

Poniżej przykład

``` python
router = APIRouter(
    tags=["Users"],
    prefix="/api/db_driver/users"
)


@router.post(path="/", 
             status_code=status.HTTP_201_CREATED,
             description="Utworzenie nowego użytkownika", 
             response_model=schemas.User, 
             response_description="User successfully created",
             responses={
                 400: {
                    "model": schemas.ErrorResponseDefault,
                    "description": "Error : Email already taken",
                    "content": {
                        "application/json": {
                            "example": {"detail": "Email already taken!"}
                        }
                    },
                }
             })
 ```
 
 Wszystkie `TODO` wypełnisz żądaniami HTTP do modułu `dbb_driver`, gdy będzie gotowy.